examples/BlazorServerApp/UserAgentContext.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace BlazorServerApp
+{
+    public class UserAgentContext
```

Comment:
Does it need to be public?

Reply:
Yes. Otherwise an error will be thrown.
Inconsistent accessibility: parameter type 'UserAgentContext' is less accessible than method 'BrowserFilter.BrowserFilter(UserAgentContext)'